frontend/util/use-query-param-state.hook.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import queryString from "query-string";
+import { identity } from "lodash-es";
+
+export function useQueryParamState(
```

Comment:
This hook allows us to put the client list search params into the frontend URL's query string. Which allows the user to refresh the page or copy/paste the URL and maintain their search parameters while on the list clients page.

Reply:
Great. This will be handy for sure.